pos_default_empty_image/README.rst

```diff
@@ -0,0 +1,96 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================================================
+Optimize loading time for products without image
+================================================
+
+This module extends the functionality of point of sale to make PoS load faster
+and to improve products display.
+
+Point Of Sale Load faster
+-------------------------
```

Comment:
Sorry I just saw it but the heading level needs to be consistent with the following ones. It should "===="
Actually you should probably add an intermediate level 1 title "introduction"